### DISCUSSION TITLE
fHow do i install libi3??

### DISCUSSION THREAD

**Discussion by rustedusted** [i3/i3#6317] [Q&A]
i'm using ubuntu 24.04 and i tried doing sudo apt install libi3 but it did not work

i just thought of writing few scripts in C to make the scripts fast unlike writing them in shell using i3-msg

**Comment by C0DE33301** [comment]
Hello!

Do you think it's "libid3-tools" not libi3?

A.R

**Comment by okraits** [comment]
You may have a look at https://i3wm.org/docs/ipc.html#_see_also_existing_libraries.
